### PR TITLE
Add Multiple nested Directory.Build.props

### PR DESCRIPTION
**Pull request by RehanSaeed**

### CODE REVIEW

File: Directory.Build.props [hunk 1]
```diff
@@ -21,6 +21,7 @@
 
   <ItemGroup Label="Package References">
     <PackageReference Include="Microsoft.CodeAnalysis.FxCopAnalyzers" PrivateAssets="All" Version="2.9.8" />
+    <PackageReference Include="Microsoft.NETFramework.ReferenceAssemblies" PrivateAssets="All" Version="1.0.0" />
```
Comment (RehanSaeed) [inline Directory.Build.props]: Should have done this in the last PR.

**Comment by RehanSaeed** [comment]
Not sure how I feel about this one. The tests is less useful I think. You could conceivably want to use a different test framework for example.

**Comment by RehanSaeed** [comment]
Tests fail when you use another `Directory.Build.props`, so removed that one.

File: Directory.Build.props (hunk 1 above)
Comment (Turnerj) [reply Directory.Build.props]: Ahhhh - yeah, missed that when I went through the files. Good idea!

**Comment by Turnerj** [comment]
I haven't had a case myself where I've wanted to mix and match test frameworks across multiple test projects in a single solution.

It does kinda suck that `Directory.Build.props` doesn't automatically import/merge from parent directories - I really thought that's how it worked.

Your changes look good to me!

**Pull request merged**